en/peripherals/uavcan_escs.md

```diff
@@ -1,20 +1,20 @@
 # UAVCAN ESCs (Motor Controllers)
 
-PX4 supports the [UAVCAN](https://uavcan.org/) bus for connecting peripherals, including ESCs, GPS modules, various types of sensors, etc.
+PX4 supports the [UAVCAN](https://uavcan.org/) bus for connecting peripherals, including ESCs, GPS modules, various types of sensors, etc. However PX4 currently does not support UAVCAN Servos.
```

Comment:
@bkueng Can you please confirm this

Reply:
Yes there's an open PR for this.

Reply:
Yes exactly. There was a user in that specific PR that was giving some feedback saying that from the current PX4 documentation he had the idea that PX4 currently supported servos, which is incorrect. So I decided to just make it a bit more clear.

Once that PR is merged (it will have to be a new one which I can hopefully create soon), then this page will have to be updated anyway.

Comment in question: https://github.com/PX4/PX4-Autopilot/pull/12984#issuecomment-764766099